grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -716,58 +900,66 @@ def _GetArraySchema(items_type_name: str) -> ArraySchema:
   }
 
 
-def _GetFieldSchema(
+def _GetMapEntryTypeName(field_name: str) -> str:
+  """Extract the name of the associated map type from a field's name."""
+  capitalized_name_components = [
+    comp.capitalize() for comp in field_name.split("_")
+  ]
```

Comment:
I find `map(str.capitalize, field_name.split("_"))` to be more elegant than list comprehensions.

Reply:
I agree. Made the change. Thank you!